src/hooks/useJurorSubscriptionFees.js

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from 'react'
+import { useWallet } from 'use-wallet'
+import { useCourtSubscriptionActions } from './useCourtContracts'
+
+export default function useJurorSubscriptionFees() {
+  const [subscriptionFees, setSubscriptionFees] = useState([])
+  const wallet = useWallet()
+  const { getters } = useCourtSubscriptionActions()
+
+  useEffect(() => {
+    const fetchSubscriptionFees = async () => {
+      if (!getters) {
+        return
+      }
+
+      try {
+        const currentPeriodId = await getters.getCurrentPeriodId()
+
+        const jurorSubscriptionsFees = []
+        for (let periodId = 0; periodId < currentPeriodId; periodId++) {
```

Comment:
Just wondering if this is going to be _really_ expensive after a while, if the periodId is unboundedly increasing.

If it is, this sounds like perhaps something for us to cache in the subgraph instead? (E.g. keeping track of which periods are relevant for a juror)

Reply:
As discussed offline with @facuspagnuolo, we'll be able to keep track of period Ids where jurors have already claimed so we can skip the `hasJurorClaimed` contract call, see https://github.com/aragon/court-subgraph/issues/48.

It would also be useful to have available in the subgraph the `periodDuration` configuration so we can re-calculate every time there's a new period.

The only contract call that we cannot bypass is the `getJurorShare` call since there's no way to save this information in the subgraph for each juror.